app/views/helpers/configure/query.phtml

```diff
@@ -18,15 +17,53 @@
 		<div class="form-group">
 			<label class="group-name" for="name"><?= _t('conf.query.name') ?></label>
 			<div class="group-controls">
-				<input type="text" name="name" id="name" value="<?= $this->query->getName()  ?>" />
+				<input type="text" name="name" id="name" value="<?= $this->query->getName() ?>" />
+				<input type="hidden" name="query[token]" id="query_token" value="<?= $this->query->getToken() ?>" />
 			</div>
 		</div>
-		<legend><?= _t('conf.query.filter') ?></legend>
 
+		<legend><?= _t('conf.query.share') ?></legend>
+		<div class="form-group">
+			<div class="group-controls">
+				<label class="checkbox" for="shareRss">
+					<input type="checkbox" name="query[shareRss]" id="shareRss" value="1" <?= $this->query->shareRss() ? 'checked="checked"' : ''?> />
+					<?= _t('conf.query.filter.shareRss') ?>
+				</label>
+				<?php if ($this->query->sharedUrlRss() !== ''): ?>
+				<ul>
+					<li><a href="<?= $this->query->sharedUrlHtml() ?>"><?= _i('link') ?> <?= _t('conf.query.share.html') ?></a></li>
+					<li><a href="<?= $this->query->sharedUrlRss() ?>"><?= _i('link') ?> <?= _t('conf.query.share.rss') ?></a></li>
+				</ul>
+				<?php endif; ?>
+			</div>
+			<div class="group-controls">
+				<label class="checkbox" for="shareOpml">
+					<input type="checkbox" name="query[shareOpml]" id="shareOpml" value="1" <?= $this->query->shareOpml() && $this->query->safeForOpml() ? 'checked="checked"' : '' ?>
+						<?= $this->query->safeForOpml() ? '' : 'disabled="disabled"' ?> />
+					<?= _t('conf.query.filter.shareOpml') ?>
+				</label>
+				<?php if ($this->query->sharedUrlOpml() !== ''): ?>
+				<ul>
+					<li><a href="<?= $this->query->sharedUrlOpml() ?>"><?= _i('link') ?> <?= _t('conf.query.share.opml') ?></a></li>
+				</ul>
+				<?php endif; ?>
+			</div>
+			<p class="help"><?= _i('help') ?> <?= _t('conf.query.share.help') ?></a></p>
+			<p class="help"><?= _i('help') ?> <?= _t('conf.query.help') ?></a></p>
+		</div>
```

Comment:
could we add a check here, if the API is disabled?
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/09d5e4f5-017e-49ba-be06-430f42012c8d)

(btw: the text in the settings needs an update, while it is not only used for the API)

Reply:
It is indeed not only used for the mobile apps any more, but also for WebSub and shared user queries (RSS, OPML, HTML)